GitSearch/Application/SearchResult/SearchResultCell.swift

```diff
@@ -30,38 +30,65 @@ class SearchResultCell: UICollectionViewCell, UICollectionViewDelegateFlowLayout
     let subTitleBackground: UIView = {
         let view = UIView()
         view.translatesAutoresizingMaskIntoConstraints = false
-        view.backgroundColor = UIColor(red: 232/255, green: 237/255, blue: 238/255, alpha: 1.0)
-        view.layer.cornerRadius = 5
+        view.backgroundColor = UIColor.white
+        view.layer.shadowRadius = 2
+        view.layer.shadowOffset = CGSize(width: 0, height: 1)
+        view.layer.shadowColor = UIColor(red: 180/255, green: 180/255, blue: 180/255, alpha: 1.0).cgColor
+        view.layer.shadowOpacity = 0.5
         return view
     }()
     
+    let subTitleBackground2: UIView = {
```

Comment:
Please change name of this variable to reflect its purpose more clearly. Usually names such as `something2`, `something3` are quite bad because you cannot distinguish all this numbered entities.

Reply:
My bad. Just leaved it on later and forgot